slyd/dist/start.html

```diff
@@ -6,10 +6,10 @@
 		html {
 		   width:100%;
 		   height:100%;
-		   background:url(/assets/portia-e34bb3dedb663765a8a50c116b0e0107.png) center center no-repeat;
+		   background:url(/portia-e34bb3dedb663765a8a50c116b0e0107.png) center center no-repeat;
```

Comment:
Left over from rebasing, I try to never pull compiled assets changes since it will prevent the pull request from merging cleanly if anybody else compiles them in the branch I want to merge in.